permits/templates/geometrywidget/geometrywidget.html

```diff
@@ -34,12 +34,16 @@
          <i class="fa fa-trash"></i>  Effacer toutes les entités
       </a>
       <a class="btn btn-light btn-sm"
-         href="javascript:geometryWidget.removeSelectedEntity();" role="button">
-         <i class="fa fa-trash"></i> Cliquer sur une entité pour la supprimer
+         href="javascript:geometryWidget.removeSelectedFeatures();" role="button">
```

Comment:
Should only be visible if entities are selected and move explanations in title tag so that only trash icon is visible